index.html

```diff
@@ -9,7 +9,7 @@
 <body>
     <h1>Red Team</h1>
 
-    <blockquote> "Comparison is the thief of joy" -Theodore Roosevelt </blockquote>
```

Comment:
Ambrose: Correct author of quote is Theodore Roosevelt not Justin Bieber. Source: Google